src/js/bootstrap-slider.js

```diff
@@ -1342,6 +1355,23 @@
 
 				return false;
 			},
+			_touchmove: function(ev) {
+				var touch = ev.changedTouches[0];
```

Comment:
The TravisCI server is breaking on the tests at this line. Here is the error: https://travis-ci.org/seiyria/bootstrap-slider/builds/120095399#L293-L295

![image](https://cloud.githubusercontent.com/assets/437318/14229248/c15eb57a-f8fc-11e5-8b69-cda69839da96.png)

Could you add a check in your code for when `ev.changedTouches[0]` is `undefined`?